Dockerfile.dev

```diff
@@ -16,7 +16,7 @@ RUN \
   mkdir /root/.tahoe-introducer; \
   mkdir /root/.tahoe-server;
 RUN /root/tahoe-lafs/venv/bin/tahoe create-introducer --location=tcp:introducer:3458 --port=tcp:3458 /root/.tahoe-introducer
-RUN /root/tahoe-lafs/venv/bin/tahoe start /root/.tahoe-introducer
+RUN /root/tahoe-lafs/venv/bin/tahoe run /root/.tahoe-introducer
```

Comment:
Yeah, @ewanas if you loot at the docs on [running tahoe](https://tahoe-lafs.readthedocs.io/en/tahoe-lafs-1.15.0/running.html) you would find that : 

> (Note that “tahoe run .” doesn’t work for introducers, this is a known issue: #937.)

However @exarkun is ticket [#937](https://tahoe-lafs.org/trac/tahoe-lafs/ticket/937) something we would want to address at this time?

Is that still relevant?

Reply:
That ticket is closed. A quick look makes me believe `tahoe run` works fine for introducers.